### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN031.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#78] [General]
# https://blog.3place.cn/3place/lmyn/LMYN031.html

https://blog.3place.cn/3place/lmyn/LMYN031.html

**Comment by picklecai** [comment]
遮那竭将要举行大祭；
这大祭十分虔诚隆重，
我们就要到那里去。 （1.30.6）

虎般的人！现在呀，
你也同我们一起登程；

兄弟俩现在荣任祭典左右护法。

**Comment by picklecai** [comment]
这大祭十分虔诚隆重，
我们就要到那里去。 （1.30.6）

虎般的人！现在呀，
你也同我们一起登程；
你将会在那里看到
那一张弓中的宝弓。 （1.30.7）

预告一下前面会掉什么装备，这样有积极性一些。

**Comment by picklecai** [comment]
每到一地，就问这是什么地方。这也是人的好奇心，也是读者的需要。
西游记里，唯一一次唐僧知道路的，就是菩提树。

**Comment by picklecai** [comment]
罗摩：听话，好奇。真是标准好学生。
